app/assets/stylesheets/application.css

```diff
@@ -128,6 +128,8 @@ form {
   background-color: #f2f2f2;
   padding: 20px;
   width: 60%;
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
Hmm, was this change something we did?  I don't recall this one, but maybe that was some solo work somewhere.  I'm sure it's fine, but I wanna double check.

Reply:
Yup, it was a change from the last PR I did that Priya merged. I saw that the site is centering headers so the forms looked funky being left aligned.

Reply:
Nice!